src/go/rdctl/cmd/snapshot.go

```diff
@@ -2,14 +2,17 @@ package cmd
 
 import (
 	"encoding/json"
+	"errors"
 	"fmt"
 	"os"
 
+	"github.com/rancher-sandbox/rancher-desktop/src/go/rdctl/pkg/snapshot"
 	"github.com/spf13/cobra"
 )
 
 type errorPayloadType struct {
-	Error string `json:"error,omitempty"`
+	Error     string `json:"error,omitempty"`
+	DataReset bool   `json:"dataReset,omitempty"`
```

Comment:
It would be good to have documentation on what the fields mean; in particular, `dataReset` isn't quite as obvious.